package.json

```diff
@@ -16,7 +16,8 @@
     "test-watch": "npm run test -- --watch",
     "lint": "eslint src/ test/",
     "lint-watch": "esw -w src/ test/",
-    "cover": "cross-env NODE_ENV=test nyc npm test"
+    "cover": "cross-env NODE_ENV=test nyc npm test",
+    "postinstall": "npm run build"
```

Comment:
Right, but when you pull down the project and install deps to start working on it, you would run `npm run dev`. I don't think we should put a `postinstall` there. 😄